corehq/warehouse/models/dimensions.py

```diff
@@ -56,8 +86,14 @@ class LocationDim(BaseDim):
     location_last_modified = models.DateTimeField()
     location_created_on = models.DateTimeField()
```

Comment:
still haven't figured out how to best model that relationship so i haven't done it yet